vumi_http_proxy/test/test_servicemaker.py

```diff
@@ -11,16 +11,26 @@ def test_defaults(self):
         options.parseOptions([])
         self.assertEqual(options["port"], 8080)
         self.assertEqual(str(options["interface"]), "0.0.0.0")
-        self.assertEqual(str(options["blacklist"]), "./docs/proxy_blacklist.yml")
+        self.assertEqual(str(options["blacklist"]), 'None')
```

Comment:
Can we remove the calls to `str` here and just assert that the blacklist is `None` and the interface is `"0.0.0.0"`?